### ISSUE TITLE
Support for the openjourney-v4 model

### ISSUE THREAD

**Issue by KixPanganiban** [brycedrennan/imaginAIry#306]
It would be amazing to support [prompthero's openjourney-v4](https://huggingface.co/prompthero/openjourney-v4). I naively thought I can grab an arbitrary model from HuggingFace by passing the `--model` parameter, ex. `--model openjourney-v4` or `--model prompthero/openjourney-v4` but that throws `ValueError: Unknown tensorfile type`

**Comment by xzitlou** [comment]
Hello, I also tried adding OJ4, I added the following in `imaginairy/config.py` and it looks like is working fine
```
MODEL_CONFIGS = [
...
ModelConfig(
        description="OpenJourney V4",
        short_name="openjourney-v4",
        config_path="configs/stable-diffusion-v1.yaml",
        weights_url="https://huggingface.co/prompthero/openjourney/resolve/main/mdjrny-v4.ckpt",
        default_image_size=512,
        default_negative_prompt="",
        alias="oj4",
    ),
]
```

**Comment by brycedrennan** [comment]
Thanks @xzitlou 

Added in just released 12.0.0
